SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -1462,6 +1464,11 @@ int32 USpatialNetDriver::ServerReplicateActors(float DeltaSeconds)
 	check(SpatialConnection && SpatialConnection->bReliableSpatialConnection);
 	check(World);
 
+	if (UReplicationDriver* RepDriver = GetReplicationDriver())
```

Comment:
Could we call `RepDriver->ServerReplicateActors(DeltaSeconds);` conditionally in `void USpatialNetDriver::TickFlush(float DeltaTime)` instead?

Reply:
It's there to resemble UE4 code, resolving.

Reply:
Followed ordering in `NetworkDriver.cpp` - will move the check